packages/cli/generators/relation/base-relation.generator.js

```diff
@@ -0,0 +1,207 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const ast = require('ts-morph');
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const path = require('path');
+const relationUtils = require('./utils.generator');
+const utils = require('../../lib/utils');
+
+module.exports = class BaseRelationGenerator extends ArtifactGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  _setupGenerator() {
+    super._setupGenerator();
```

Comment:
Can you move the code from `_initArtifacts` to `_setupGenerator` before the call to `super._setupGenerator()` similar to https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/controller/index.js#L30 and https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/model/index.js#L44?

Reply:
Fixed.